itou/siae_evaluations/fixtures.py

```diff
@@ -0,0 +1,136 @@
+Employer's account: test+etti@inclusion.beta.gouv.fr
+Labor inspector's account: test+ddets@inclusion.beta.gouv.fr
+"""
+
+import random
+from unittest import mock
+
+import django
+from dateutil.relativedelta import relativedelta
+from django.test import override_settings
+from django.utils import timezone
+
+
+if __name__ == "__main__":
+    django.setup()
+
+    import itou.siae_evaluations.enums as evaluation_enums
+    import itou.users.enums as users_enums
+    from itou.approvals.models import Approval
+    from itou.eligibility.enums import AdministrativeCriteriaLevel
+    from itou.eligibility.models import AdministrativeCriteria
+    from itou.eligibility.models.iae import EligibilityDiagnosis
+    from itou.institutions.models import Institution
+    from itou.job_applications.models import JobApplication, JobApplicationWorkflow
+    from itou.siae_evaluations.factories import EvaluationCampaignFactory
+    from itou.siae_evaluations.models import EvaluationCampaign
+    from itou.siaes.models import Siae
+    from itou.users.models import User
+
+    NAMES = [
+        ("Gustave", "Loiseau", "187104712403340", "703059J"),
+        ("Barnabé", "Loiseau", "187105714612783", "476261C"),
+        ("Oliveria", "Da Figueira", "187102017128676", "354471C"),
+        ("Fan", "Se-Yeng", "187102531808302", "145793B"),
+        ("Diego", "le Navarrais", "187101703338725", "718757J"),
+        ("Bianca", "Castafiore", "287103404102922", "50704F"),
+        ("Tryphon", "Tournesol", "187105012012584", "321089E"),
+        ("Bunji", "Kuraki", "187101300806442", "298096C"),
+        ("Roberto", "Rastapopoulos", "187107922316551", "424548G"),
+        ("Séraphin", "Lampion", "187104308500783", "491405D"),
+        ("Allan", "Thompson", "187101033733471", "391055C"),
+        ("Eugène", "Triboulet", "187107317012993", "103819B"),
+        ("Igor", "Wagner", "187107823832946", "621359J"),
+        ("Frank", "Wolff", "187106129833161", "644055J"),
+        ("Tristan", "Bior", "187106539602337", "355357E"),
+        ("Herbert", "Dawes", "187103415217885", "401486E"),
+        ("Émile", "Vanneau", "187102117208427", "354895F"),
+        ("Jean-Loup", "de La Batellerie", "187109413107670", "354895D"),
+    ]
+
+    def generate_approval_number():
+        return f"{Approval.ASP_ITOU_PREFIX}{random.randint(0000000, 9999999)}"
+
+    with override_settings(EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
+        today = timezone.localdate()
+        evaluated_period_start_at = today - relativedelta(months=3)
+        evaluated_period_end_at = today - relativedelta(months=1)
+        created_job_applications = []
+        controlled_siae = Siae.objects.get(pk=2653)  # has an active convention
+        siae_first_member = controlled_siae.members.first()
+        total_administrative_criteria = AdministrativeCriteria.objects.count()
+        criteria_pks = AdministrativeCriteria.objects.values("pk", "level")
```

Comment:
J'aurais fait:
```suggestion
        level_to_critera_pks = dict(AdministrativeCriteria.objects.values("level").annotate(pks=ArrayAgg('pk')).values_list("level", "pks"))
```
(avec un `from django.contrib.postgres.aggregates import ArrayAgg`)